dropwizard-configuration/src/main/java/io/dropwizard/configuration/ResourceConfigurationSourceProvider.java

```diff
@@ -6,6 +6,11 @@
 public class ResourceConfigurationSourceProvider implements ConfigurationSourceProvider {
     @Override
     public InputStream open(String path) throws IOException {
+        InputStream result = getResourceAsStream(path);
+        return result == null && path.startsWith("/") ? getResourceAsStream(path.substring(1)) : result;
```

Comment:
How about a Javadoc comment on the class, since there currently isn't one?

```
/**
 * A {@link ConfigurationSourceProvider} which reads the configuration
 * from a resource file.
 *
 * In order to abide by the calling conventions of
 * {ClassLoader#getResourceAsStream} [1], absolute path strings
 * (i.e. those with leading "/" characters) passed to {@link #open(String)}
 * are converted to relative paths by removing the leading "/".
 *
 * See [1] for more information on resources in Java and how they are
 * loaded at runtime.
 *
 * [1] https://docs.oracle.com/javase/8/docs/technotes/guides/lang/resources.html
 */
public class ResourceConfigurationSourceProvider implements ConfigurationSourceProvider {
...
```